packages/default/scss/toolbar/_theme.scss

```diff
@@ -108,6 +155,40 @@
 
     }
 
+    // RTL
+    .k-rtl,
+    [dir="rtl"] {
+        .k-toolbar-scrollable-overlay {
+            &::before {
+                background: linear-gradient(270deg, $kendo-toolbar-scroll-overlay);
+            }
+
+            &::after {
+                background: linear-gradient(90deg, $kendo-toolbar-scroll-overlay);
+            }
+
+            &.k-toolbar-outline {
+                &::before {
+                    background: linear-gradient(270deg, $kendo-toolbar-outline-scroll-overlay);
+                }
+
+                &::after {
+                    background: linear-gradient(90deg, $kendo-toolbar-outline-scroll-overlay);
+                }
+            }
+
+            &.k-toolbar-flat {
```

Comment:
@The linear-gradient variables below should be `$kendo-toolbar-flat-scroll-overlay`